HOMEWORK2.HTML

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h2>JavaScript For Loop</h2>
+  <p id="max_number"></p>
+  <p id="min_number"></p>
+  <p id="total"></p>
+  <script>
+    const numbers = [8, 1, 2, 3, 5, 6];
+    function findmax ()
+    {
+       let a = numbers.length;
+       let max = numbers [0];
+       for (let i=0; i<a; i++)
+     {
+       if (numbers[i]>max)
+       {
+         max = numbers[i];
+       }
+     }
+      return max;
+    }
+    document.getElementById("max_number").innerHTML = findmax ();
+    function findmin()
+    {
+      let a = numbers.length
+      let min = numbers [0];
+      for (let i=0; i<a; i++)
+      {
+        if (numbers[i]<min)
+        {
+          min = numbers [i];
+        }
+      }
+      return min;
+    }
+    document.getElementById("min_number").innerHTML = findmin();
+    function total()
+    {
+      let a = numbers.length
```

Comment:
do we need to change the value of `a`? if not, why didn't we use `const` for it?